lib/src/fty_common_nut_utils_private.cc

```diff
@@ -28,44 +28,69 @@ int runCommand(
     const std::string& cmd, const Process::Arguments& args, std::string& stdout, std::string& stderr, int timeout)
 {
     std::stringstream fullCommand;
+    fullCommand << cmd << " ";
     for (const auto& i : args) {
         fullCommand << i << " ";
     }
 
     int msTimeout = timeout * 1000;
 
     std::string fullCommandStr = fullCommand.str();
-    log_info("Running command %s (with %d seconds timeout)...", cmd.c_str(), timeout);
+    logInfo("Running command {} (with {} seconds timeout)...", fullCommandStr, timeout);
 
     Process proc(cmd, args, Capture::Out | Capture::Err);
     if (auto pid = proc.run(); !pid) {
-        log_error(pid.error().c_str());
+        logError("{}", pid.error());
         return -1;
     }
 
-    auto ret = proc.wait(msTimeout);
-    if (!ret) {
-        log_error(ret.error().c_str());
-        return -1;
+    const int WAIT_TIMEOUT_LOOP = 2000;
+    int count = msTimeout / WAIT_TIMEOUT_LOOP;
```

Comment:
shall be: int count = ((msTimeout / WAIT_TIMEOUT_LOOP) + 1);
with the if(count)...